packages/common/src/search/_internal/hubSearchItemsHelpers/ogcApiRequest.ts

```diff
@@ -30,13 +30,16 @@ import { getQueryString } from "./getQueryString";
 export async function ogcApiRequest(
   url: string,
   queryParams: Record<string, any>,
-  options: IHubSearchOptions
+  options: ISearchOgcItemsOptions
 ) {
   const updatedQueryParams = cloneObject(queryParams);
 
-  const targetDomain = new URL(options.site).hostname;
+  let targetDomain;
+  if (options.site) {
+    targetDomain = new URL(options.site).hostname;
+  }
   const urlDomain = new URL(url).hostname;
-  if (targetDomain !== urlDomain) {
+  if (targetDomain && targetDomain !== urlDomain) {
```

Comment:
We ultimately want to deprecate the `options.site` property, but can't do it yet. As first step, we're making it (and by extension the target domain) optional